### PR TITLE
updated English Courses

### PR DESCRIPTION
**Pull request by apu52**
## What does this PR do?
Add resource(s)

### Description
I have added a Software Engineering course that is useful for all the beginners as well as for the professionals
### Why is this valuable (or not)?
This is very valuable
### How do we know it's really free?
This is totally free as it's available on YouTube
### For book lists, is it a book? For course lists, is it a course? etc.
It's a course
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by apu52** [comment]
image

I have added my new C course in this English courses section, But the error is showing there where I haven't done anything.

**Comment by apu52** [comment]
@Ajay-singh1 @eshellman Please take a review of my PR & please merge it.

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -152,6 +152,7 @@
 * [C Language Tutorial for Beginners (with Notes & Practice Questions)](https://www.youtube.com/watch?v=irqbmMNs2Bo) - Apna College
 * [C Language Tutorial For Beginners In Hindi (With Notes)](https://www.youtube.com/watch?v=ZSPZob_1TOk) - CodeWithHarry
 * [C Language Tutorial in Hindi](https://www.youtube.com/playlist?list=PLmRclvVt5DtksgReOH3s7R1_cb1QA8vrb) - codeitup
+* [C Language Tutorial in Hindi](https://www.youtube.com/playlist?list=PLmRclvVt5DtksgReOH3s7R1_cb1QA8vrb) - codeitup
```
Comment (eshellman) [inline courses/free-courses-hi.md:155]: duplicate

**Comment by apu52** [comment]
@eshellman I have removed the duplicate link & also have added a good software engineering course in English courses section. Can you please take a review & please merge it.

**Pull request merged**